src/util/integer_gmp_imp.cpp

```diff
@@ -340,6 +340,21 @@ bool Integer::isNegativeOne() const
 }
 
 Integer Integer::pow(unsigned long int exp) const
```

Comment:
Please make it a `uint64_t`

Reply:
Also here? https://github.com/cvc5/cvc5/blob/85ab7cba7ebf1fa989777f074cb850949f861e93/src/util/integer_gmp_imp.h#L208
and here and here?
https://github.com/cvc5/cvc5/blob/master/src%2Futil%2Finteger_cln_imp.h#L210
https://github.com/cvc5/cvc5/blob/85ab7cba7ebf1fa989777f074cb850949f861e93/src/util/integer_cln_imp.cpp#L274

Reply:
yes. There, it should be purely cosmetic as it is directly converted to a `cl_I`.